images/kubevirt-infra-bootstrap/Dockerfile

```diff
@@ -16,3 +16,14 @@ RUN apt-get update \
         python3 \
         python3-pip \
         && rm -rf /var/lib/apt/lists/*
+RUN export GOROOT=/opt/go/root \
+    && mkdir -p $GOROOT \
+    && export GOPATH=/opt/go/path \
+    && mkdir -p $GOPATH \
+    && export PATH=$GOPATH/bin:$GOROOT/bin:$PATH \
+    && export GIMME_GO_VERSION=1.12 \
+    && GIMME=/opt/go/gimme \
+    && mkdir -p $GIMME \
+    && curl -sL https://raw.githubusercontent.com/travis-ci/gimme/master/gimme | HOME=${GIMME} bash >> ${GIMME}/gimme.sh \
+    && . ${GIMME}/gimme.sh \
+    && GO111MODULE=on go get sigs.k8s.io/kustomize
```

Comment:
just curious, why not just downloading kustomize from its release page? Installing Go just for this seems quite some overhead to me :) 
https://github.com/kubernetes-sigs/kustomize/releases/tag/kustomize%2Fv3.5.4

Reply:
Totally agree. Thanks!